packages/fast-tooling-react/src/css-property-editor/property-editor.spec.tsx

```diff
@@ -144,4 +137,43 @@ describe("CSSPropertyEditor", () => {
             shallow(<CSSPropertyEditor data={data} />);
         }).not.toThrow();
     });
+
+    test("row with empty key gets delected on input blur", () => {
+        const data: { [key: string]: string } = {
+            padding: "10px",
+            margin: "30px",
+        };
+        const emptyKey: string = "";
+        const rendered: any = mount(<CSSPropertyEditor data={data} />);
+        let inputs: any = rendered.find("input");
+        expect(inputs).toHaveLength(4);
+        inputs.at(0).simulate("focus");
+        inputs.at(0).simulate("change", { target: { value: emptyKey } });
+        inputs.at(0).simulate("blur");
+
+        inputs = rendered.find("input");
+        expect(inputs).toHaveLength(2);
+    });
+
+    test("edited key values get committed on enter key press", () => {
```

Comment:
can you change this text to "should fire the onChange callback to add new key values when the enter key is pressed"

Reply:
yep